src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSparkUnitTest.java

```diff
@@ -95,12 +96,40 @@ public void writeToLocalTest() throws IOException{
         Assert.assertTrue( FileUtils.contentEqualsIgnoreEOL(copied._2(), originalFASTQFile, StandardCharsets.UTF_8.name()) );
     }
 
+    // Test if disabling stdout stderr works
+    @Test(groups = "sv")
+    public void stdioCaptureTest() throws IOException{
```

Comment:
Now done with added test, though admittedly `stdout` is not captured (the process doesn't produce stdout results); but stderr is indeed captured correctly.